doc/flame/components.md

```diff
@@ -70,12 +70,12 @@ A component lifecycle state can be checked by a series of getters:
 ### Priority
 
 In Flame every `Component` has the `int priority` property, which determines
-that component's sorting order within its parent's children, this is sometimes referred to
+that component's sorting order within its parent's children; this is sometimes referred to
 as `z-index` in other languages and frameworks. The higher the `priority` is set to, the
 closer the component will appear on the screen, since it will be rendered on top of any components
 with lower priority that were rendered before it.
 
-If you add two components and set one of them to priority 1 for example, then that component will be
+If you add two components and set one of the priorities to 1 for example, then that component will be
```

Comment:
the or their is correct.  Its just preference.  Changed to their.

Reply:
It's grammatically correct, but it doesn't mean the same thing. "their" is more strict since it refers to the subject, meanwhile "the" can refer to any priorities.